src/traceur.js

```diff
@@ -15,12 +15,11 @@
 // Used by unit tests only
 import './util/MutedErrorReporter.js';
 
-export {ModuleStore} from '@traceur/src/runtime/ModuleStore.js';
 export {WebPageTranscoder} from './WebPageTranscoder.js';
 export {HTMLImportTranscoder} from './HTMLImportTranscoder.js';
 import {addOptions, CommandOptions, Options} from './Options.js';
 
-import {ModuleStore} from '@traceur/src/runtime/ModuleStore.js';
+let ModuleStore = $traceurRuntime.ModuleStore;
```

Comment:
Added
```
// The ModuleStore has to be instantiated before we can register any modules.
// That means our normal module registration, System.registerModule() 
// cannot be applied to ModuleStore. We can use System.set() to store the
// ModuleStore module, but that function does not work for --modules=inline.
```
The complete explanation would include: why do we need `ModuleStore` in `src/traceur.js`? To allow tests to look up modules by name rather than import them.  But since I changed the unit tests over to es6 the only remaining uses of this secret may be tests of `System.set()`.